gap_fit_module.f95

```diff
@@ -210,7 +213,23 @@ subroutine gap_fit_parse_command_line(this)
      mpi_blocksize => this%mpi_blocksize
      
      call initialise(params)
-     
+
+     call param_register(params, 'config_file', '', config_file, has_value_target=has_config_file, &
+          help_string="File as alternative input (newlines converted to spaces)")
+
+     ! check if config file is given, ignore everything else
+     ! prepare parsing of config file or command line string later
+     if (param_read_args(params, ignore_unknown=.true., command_line=this%command_line)) then
+        if (has_config_file) then
+           inquire(file=config_file, exist=file_exists)
+           if (.not. file_exists) call system_abort("Config file does not exist: "//config_file)
+           call read(config_str, config_file, keep_lf=.false.)
+           string_to_parse = string(config_str)
```

Comment:
I think it would be better to make this conversion when passing it to `param_read_line()`. That way the string size is not limited by the string length (only by the stack size).

Reply:
Good idea, should work.